fields.yml

```diff
@@ -982,19 +980,28 @@
             Sometimes called program name or similar.
           example: ssh
     
-        - name: pid
-          level: core
-          type: long
-          description: >
-            Process id.
-          exmple: ssh
-    
         - name: ppid
           level: extended
           type: long
           description: >
             Process parent id.
     
+        - name: args
+          level: extended
+          type: keyword
+          description: >
+            Process arguments.
+    
+            May be filtered to protect sensitive information.
+          example: ["-l", "user", "10.0.0.16"]
+    
+        - name: executable
+          level: extended
+          type: keyword
+          description: >
+            Full path to the process executable.
```

Comment:
How about using the word "Absolute" rather than "Full"? They're the same thing, but IMO "absolute path" is even less ambiguous.

Reply:
Good point.